flow360/component/simulation/models/solver_numerics.py

```diff
@@ -149,22 +170,52 @@ class NavierStokesSolver(GenericSolverSettings):
     >>> ns = NavierStokesSolver(absolute_tolerance=1e-10)
     """
 
-    absolute_tolerance: PositiveFloat = pd.Field(1.0e-10)
+    absolute_tolerance: PositiveFloat = pd.Field(
+        1.0e-10,
+        description="Tolerance for the NS residual, below which the solver goes to the next physical step",
+    )
 
-    CFL_multiplier: PositiveFloat = pd.Field(1.0)
-    kappa_MUSCL: pd.confloat(ge=-1, le=1) = pd.Field(-1)
+    CFL_multiplier: PositiveFloat = pd.Field(
+        1.0, description="Factor to the CFL definitions defined in :code:`Time stepping` section"
```

Comment:
:code:`Time stepping` does not seem to work? 
![image](https://github.com/user-attachments/assets/7570deab-8067-4ba4-bc63-e41e18ecc60a)
Also can you delete the 
```
Parameters
----------
```
part? It is causing duplication:

![image](https://github.com/user-attachments/assets/4e759a24-ad89-422d-a8a6-45fa82645131)